bindata/etcd/minimal-sm.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: etcd
+  namespace: openshift-etcd-operator
```

Comment:
don't know if it's ok to deploy resources on operator namespace
(still think this makes more sense on `openshift-etcd` ns, but your call)

Reply:
absolutely fine